packages/wasmtime/wasmtime.0.0.1/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+name:         "wasmtime"
+bug-reports:  "https://github.com/LaurentMazare/ocaml-wasmtime/issues"
+homepage:     "https://github.com/LaurentMazare/ocaml-wasmtime"
+dev-repo:     "git+https://github.com/LaurentMazare/ocaml-wasmtime.git"
+maintainer:   "Laurent Mazare <lmazare@gmail.com>"
+authors:      [ "Laurent Mazare" ]
+
+build: [["dune" "build" "-p" name "-j" jobs]]
+
+run-test: [
+  ["dune" "runtest" "-p" name "-j" jobs] { os-distribution != "alpine" }
+]
+
+depends: [
+  "base" {>= "v0.13.0" & < "v0.15"}
+  "ctypes" {>= "0.5"}
+  "ctypes-foreign"
+  "dune" {>= "2.0.0"}
```

Comment:
Almost there, the dune-project file in the repo requires dune 2.7
```suggestion
  "dune" {>= "2.7.0"}
```

Reply:
Thanks for catching this!